### PR TITLE
Remove deprecated call in FastjetJetProducer [to be able to update to fastjet-contrib 1.026]

### PR DESCRIPTION
**Pull request by rappoccio**
This is a trivial addition to a request for fastjet-contrib's update to 1.026. Here I turned off some deprecated code.

### CODE REVIEW

**Comment by slava77** [comment]
@rappoccio 
please edit the PR title to be more specific to what you do in addition to why

The change is better titled smth like "Remove deprecated call in FastjetJetProducer [to be able to update to fastjet-contrib 1.026]"
Thank you.

**Comment by slava77** [comment]
@cmsbuild please test

**Comment by rappoccio** [comment]
OK done!

**Comment by rappoccio** [comment]
This will subsequently be integrated at the same time as this PR:

https://github.com/cms-externals/fastjet-contrib/pull/11

**Comment by iahmad-khan** [comment]
please test with cms-sw/cmsdist#2959

File: RecoJets/JetProducers/plugins/FastjetJetProducer.cc [hunk 1]
```diff
@@ -443,7 +443,7 @@ void FastjetJetProducer::runAlgorithm( edm::Event & iEvent, edm::EventSetup cons
       bge_rho->set_particles(fjInputs_);
       fastjet::contrib::ConstituentSubtractor * constituentSubtractor = new fastjet::contrib::ConstituentSubtractor(bge_rho.get());
       // this sets the same background estimator to be used for deltaMass density, rho_m, as for pt density, rho:
-      constituentSubtractor->use_common_bge_for_rho_and_rhom(true); // for massless input particles it does not make any difference (rho_m is always zero)
+      //constituentSubtractor->use_common_bge_for_rho_and_rhom(true); // for massless input particles it does not make any difference (rho_m is always zero)
```
Comment (slava77) [inline RecoJets/JetProducers/plugins/FastjetJetProducer.cc]: is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant

**Comment by slava77** [comment]
@rappoccio 
your action is needed here before the PR can move on.
Thank you.

File: RecoJets/JetProducers/plugins/FastjetJetProducer.cc (hunk 1 above)
Comment (rappoccio) [inline RecoJets/JetProducers/plugins/FastjetJetProducer.cc]: Sorry for the delay (was on holiday this weekend). The code is not needed. Removed now.

Comment (slava77) [inline RecoJets/JetProducers/plugins/FastjetJetProducer.cc]: an update commit didn't show up yet in this PR

**Comment by slava77** [comment]
@rappoccio 
please clarify on the status.
Currently this blocks the fastjet external integration.

File: RecoJets/JetProducers/plugins/FastjetJetProducer.cc (hunk 1 above)
Comment (rappoccio) [inline RecoJets/JetProducers/plugins/FastjetJetProducer.cc]: Sorry Slava, pushed to the wrong branch. Should be fixed now.

**Comment by slava77** [comment]
@cmsbuild please test

**Comment by slava77** [comment]
+1

for https://github.com/cms-sw/cmssw/pull/18351	4a4ad60
- as described
- jenkins tests pass (based on commented out line in the earlier commit) and comparisons with baseline show no differences

**Pull request merged**